kpatch-build/kpatch-build

```diff
@@ -287,7 +293,7 @@ mkdir -p "$TEMPDIR" || die "Couldn't create $TEMPDIR"
 rm -rf "$TEMPDIR"/*
 rm -f "$LOGFILE"
 
-trap cleanup EXIT INT TERM HUP
+[[ $SKIPCLEANUP -eq 0 ]] && trap cleanup EXIT INT TERM HUP
```

Comment:
Actually, scratch that comment. It'd be useful to keep $TMPDIR/$RPMTOPDIR under those other conditions.